dev/devicelab/lib/tasks/perf_tests.dart

```diff
@@ -67,6 +67,12 @@ TaskFunction createGalleryBackButtonMemoryTest() {
   );
 }
 
+TaskFunction createFlutterViewStartupTest() {
+  return new StartupTest(
```

Comment:
The `StartupTest` will publish start-up metrics to [the dashboard](https://flutter-dashboard.appspot.com/benchmarks.html). I'm not sure these metrics will be useful? Can we add a `reportMetrics` to `StartupTest`'s constructor, default it to `true` and set it to `false` for this test?